Userland/Libraries/LibWeb/Painting/PaintContext.h

```diff
@@ -46,6 +46,16 @@ class PaintContext {
         return m_svg_transform;
     }
 
+    bool draw_svg_geometry_for_clip_path() const
+    {
+        return m_draw_svg_geometry_for_clip_path;
```

Comment:
"The all style properties" in commit desc

Reply:
Fixed :+1: